src/rules/store/use-selector-in-select.ts

```diff
@@ -23,13 +23,13 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
     docs: {
       category: 'Best Practices',
       description:
-        'Using a selector in a select method is preferred in favor of strings or props drilling.',
+        'Using a selector in a `select` method is preferred over `string` or `props drilling`.',
```

Comment:
I was copying (😅) this description to #248 and realized that "over" fits better than "in favor of" in this case, so I decided to change this here as well. Let me know if it's fine for you too.

Reply:
Yep, this is better 👍